components/atoms/ViewMoreLessButton.js

```diff
@@ -4,7 +4,7 @@ import ActionButton from './ActionButton'
 export default function ViewMoreLessButton(props) {
   return (
     <ActionButton
-      className="text-xl leading-8 py-5 pl-2 md:pl-6 underline text-link-blue-default hover:text-link-blue-hover"
+      className="text-xl leading-8 py-5 pl-2 md:pl-6 text-link-blue-default hover:text-link-blue-hover"
```

Comment:
I believe this would make that button/link inaccessible... (it's a bit subjective, so maybe we should check with an a11y expert)
https://www.w3.org/WAI/WCAG21/Techniques/failures/F73
> Check that each link in the page that is identifiable by color (hue) is visually identifiable via some other means (e.g., underlined, bolded, italicized, sufficient difference in lightness, etc).